scripts/util/detect-changed-contracts.js

```diff
@@ -40,6 +51,13 @@ async function detectChangedContract(referenceCommit, targetCommit) {
   shell.exec(`rm -rf contracts`);
   shell.exec(`git checkout ${referenceCommit} contracts`);
 
+  // Protocol versions < 2.3.0 use different OZ contracts. We need to install them
+  const isOldVersion = ["v2.0", "v2.1", "v2.2"].some((v) => referenceCommit.startsWith(v));
+  if (isOldVersion) {
+    // Temporary install old OZ contracts
+    shell.exec("npm i @openzeppelin/contracts-upgradeable@4.7.1");
+  }
+
```

Comment:
Although this looks fine since OZ is the only library we are importing on our contracts, I would prefer to run `npm install` and run it every time (not only to upgrade OZ) to turn this more generic since we can use other libraries in the future and also to make sure that we are testing the expected code

Reply:
(but I'm happy if you rather to keep this way)

Reply:
Makes sense. Changed.